layers/slices.go

```diff
@@ -65,6 +65,7 @@ func (f *Factory) createLayerFromSlice(slice Slice, sdir *sliceableDir, layerID
 }
 
 func glob(sdir *sliceableDir, pattern string) ([]string, error) {
+	pattern = filepath.Clean(pattern)
```

Comment:
Maybe it would be better to clean the slices when `metadata.toml` is read (or written). But this seemed like the smallest & safest change.

Reply:
thanks!